testsuite/tests/navigation/generic_pkg_top_level_2/foo.adb

```diff
@@ -0,0 +1,3 @@
+package body Foo is
```

Comment:
No test directives?

Reply:
Nevermind :D, that's good actually.

Reply:
This is only to test the `p_body_part_for_decl` property, so the directive is in the `test.adb` file.